docs/content/docs/recipes/signals.md

```diff
@@ -0,0 +1,516 @@
+---
+title: Using Signals in Fluid
+menuPosition: 5
+draft: true
+---
+
+In this recipe, we will go over the `PresenceTracker` example to learn how the `Signaler` DataObject is used in a Fluid application to deal with a couple user presence scenarios. The two forms of presence covered in this example are focus tracking and mouse tracking. This article will go over how the `Signaler` DataObject is used in both the `MouseTracker` and `FocusTracker` classes to achieve user presence in a Fluid application. You can find the completed code for this example in the Fluid Repo [here](https://github.com/microsoft/FluidFramework/tree/main/examples/data-objects/presence-tracker).
+
+## Creation
+
+The `Signaler` DataObject is first included in the `initialObjects` of the Fluid `ContainerSchema` which defines which shared objects will be at our disposal in the new container.
+
+```typescript
+const containerSchema: ContainerSchema = {
+    initialObjects: {
+        /* [id]: DataObject */
+        signaler: Signaler,
+    },
+};
+```
+
+The `FluidContainer` is then created (or fetched if it already exists) alongside the `services` object, since audience will be required in this application. To learn more about how to create a new Fluid container and load an existing container, see our [DiceRoller Tutorial](https://fluidframework.com/docs/start/tutorial/).
+
+## MouseTracker
+
+Let's look at how the `MouseTracker` class uses the `Signaler` DataObject to achieve the first form of user presence within the application.
+
+The class defines the `mouseSignalType` that will be sent and listened to the connected clients when there is a presence change:
+
+```typescript
+export interface IMouseTrackerEvents extends IEvent {
+    (event: "mousePositionChanged", listener: () => void): void;
+}
+
+export class MouseTracker extends TypedEventEmitter<IMouseTrackerEvents> {
+    private static readonly mouseSignalType = "positionChanged";
+
+    /*...*/
+}
+```
+The class also initializes a local map of `IMousePosition` values for all of the connected clients. Each userID can have multiple clientIDs (e.g. same user on separate devices), which explains the nested `Map`. This local map is what populates the view and what will be updated on `mouseSignalType` signals:
```

Comment:
Maybe reference the Audience article to explain the user->client nesting:

https://fluidframework.com/docs/build/audience/